src/deprecates.jl

```diff
@@ -48,3 +48,15 @@ findat(a::AbstractArray, b::AbstractArray) = findat!(Array{Int}(size(b)), a, b)
 @deprecate df_residual(obj::StatisticalModel) dof_residual(obj)
 
 @deprecate_binding WeightVec Weights
+
+using Base.Random: RangeGenerator
+@deprecate_binding RandIntSampler RangeGenerator
+@deprecate RangeGenerator(K::Int) RangeGenerator(1:K)
```

Comment:
That syntax doesn't work. I've also tried defining a standard function and calling `depwarn` inside it, but that still triggers the deprecation warning. Anyway, `@deprecate_binding` defines `const RandIntSampler = RangeGenerator`, so in the end the method is defined for `RangeGenerator` under the hood.